src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -92,6 +92,12 @@ function MoneyRequestParticipantsSelector({participants = CONST.EMPTY_ARRAY, onF
         searchInServer(debouncedSearchTerm.trim());
     }, [debouncedSearchTerm]);
 
+    useEffect(() => {
```

Comment:
does it cause any extra rerender? Do we have a way to check this?

Reply:
Based on my testing, this just adds a function call